jwst/regtest/test_nirspec_fs_spec2.py

```diff
@@ -135,3 +136,25 @@ def test_pathloss_source_type(rtdata):
             if not np.allclose(slit.data, slit.pathloss_uniform, equal_nan=True):
                 bad_slits.append(idx)
     assert not bad_slits, f'Force to uniform failed for slits {bad_slits}'
+
+
+@pytest.mark.bigdata
+def test_short_calwebb_spec2_rateints(rtdata_module):
```

Comment:
Since this will now act as a generic test of sending a 3D FS (rateints) file through all of spec2, I would just call it "test_nirspec_fs_rateints_spec2" (in keeping with existing test names in this module).

Reply:
Done